docusaurus-plugin-moonwave/src/components/LuaType.js

```diff
@@ -181,18 +193,46 @@ function Token({ token, depth }) {
     case "union":
       return <Op>&nbsp;|&nbsp;</Op>
     case "luaType":
+      const sanitizedToken = token.luaType.replace(/\W/g, "")
+      // Checks if the type is in the list of typeLinks for the package
+      if (typeLinks.has(sanitizedToken)) {
+        // If the type is to an external Roblox default, link to the external Roblox Devhub
+        if (isValidUrl(typeLinks.get(sanitizedToken))) {
+          return (
+            <code className={styles.blue}>
+              <a
+                style={{ textDecoration: "underline", color: "inherit" }}
+                href={typeLinks.get(sanitizedToken)}
+              >
+                {token.luaType}
+              </a>
+            </code>
+          )
+        }
+        // If the type is from another page in the Moonwave package
+        else {
+          return (
+            <code className={styles.blue}>
+              <Link
+                style={{ textDecoration: "underline", color: "inherit" }}
+                to={typeLinks.get(sanitizedToken)}
+              >
+                {token.luaType}
+              </Link>
+            </code>
+          )
+        }
+      }
+
       return <code className={styles.blue}>{token.luaType}</code>
     default:
       return <span>unknown token {Object.keys(token)[0]}</span>
   }
 }
 
 export default function LuaType({ code }) {
-  if (true) {
-    const tokens = tokenize(code)
-
-    return <Token token={{ root: tokens }} />
-  }
+  const tokens = tokenize(code)
+  const typeLinks = useContext(TypeLinksContext)
```

Comment:
Why do we `useContext` here, instead of in the `Token` component? We are just passing it through here anyways, so it seems better to use the context where we actually need the data.